### PR TITLE
Update issue and PR templates

### PR DESCRIPTION
**Pull request by gamer191**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)
- I trust github actions to take care of that

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

---

### Description of your *pull request* and other information

This PR suggests a few minor changes to the issue templates, the most useful one being that the feature request template will now require the user to tick a box that their feature request is not related to any specific sites.

At some point in the near future, I will likely create a new PR with more suggestions related to the issue templates, but since these changes are unlikely to cause conflicts, I figured I may as well make a PR now, and make a new PR when I think of more suggestions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/6_question.yml [hunk 1]
```diff
@@ -9,13 +9,15 @@ body:
...
         Carefully read and work through this check list in order to prevent the most common mistakes and misuse of yt-dlp:
       options:
-        - label: I'm asking a question and **not** reporting a bug/feature request
+        - label: I'm asking a question and **not** reporting a bug or requesting a feature
           required: true
         - label: I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
           required: true
-        - label: I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
+        - label: I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
+          required: true
```
Comment (pukkandan) [inline .github/ISSUE_TEMPLATE/6_question.yml:17]: not needed for question

File: .github/ISSUE_TEMPLATE/6_question.yml [hunk 2]
```diff
@@ -9,13 +9,15 @@ body:
...
+        - label: I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones. DO NOT post duplicates
           required: true
-        - label: I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones
+        - label: I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
```
Comment (pukkandan) [inline .github/ISSUE_TEMPLATE/6_question.yml:20]: why re-order?

Comment (gamer191) [reply .github/ISSUE_TEMPLATE/6_question.yml:20]: All the other issue templates use this order. It's not really a necessary change, but I figured I may as well include it in this PR. If you don't want it I can remove it though

**Comment by gamer191** [comment]
I'm marking this as "ready for review" again. I do have more changes to suggest, but they go beyond the scope for this PR, so I'll make a new PR some time soon

**Comment by pukkandan** [comment]
I have cherry-picked the first batch of changes. The rest are undesirable

**Comment by MrRawes** [comment]
i would also suggest changing the url for "Unlicense" to https://github.com/yt-dlp/yt-dlp/blob/master/LICENSE

**Review by MrRawes — CHANGES_REQUESTED** [review]
i suggest changing the url for "Unlicense" to be https://github.com/yt-dlp/yt-dlp/blob/master/LICENSE

**Comment by gamer191** [comment]
> i suggest changing the url for "Unlicense" to be https://github.com/yt-dlp/yt-dlp/blob/master/LICENSE

Personally I'm not a fan of that idea. What do you think, Pukkandan?

That said, the URL should definitely be changed from http to https

**Comment by pukkandan** [comment]
Technically, `http` is right

image

None of these changes even matter anyway!

**Comment by fstirlitz** [comment]
A honeypot item on the checklist?

> - [ ] I fail at reading comprehension and following written instructions, and I want to be banned from the tracker

GitHub Action reply: ‘Your wish is granted.’ **\*BANHAMMER\***

Plus, randomize the order of checklist items to keep people on their toes.

**Comment by gamer191** [comment]
> None of these changes even matter anyway!

Fair call!

File: .github/ISSUE_TEMPLATE/6_question.yml (hunk 1 above)
Comment (pukkandan) [reply .github/ISSUE_TEMPLATE/6_question.yml]: I take this back. There are issues posted where this would have been needed

**Pull request closed without merging**